app/views/skins/confirm_delete.html.erb

```diff
@@ -0,0 +1,13 @@
+<!--Descriptive page name, messages and instructions-->
+<h2 class="heading"><%= ts("Delete Skin") %></h2>
+
+<!--main content-->
+<%= form_for(@skin, html: { method: :delete, class: "simple destroy" }) do |f| %>
+  <p class="caution notice">
+    <%= t("skins.confirm_delete_html", skin_title: @skin.title) %>
```

Comment:
One last thing I just noticed, we can use [lazy key lookup](https://guides.rubyonrails.org/i18n.html#lazy-lookup), and keep the locale file organized similar to the views folder. Can you change the lookup to be:
```erb
<%= t(".confirm_html", skin_title: @skin.title) %>
```
and the locale file to be like this?
```yml
  skins:
    confirm_delete:
      confirm_html: Are you sure you want to <strong><em>delete</em></strong> the skin \"%{skin_title}\"?
```